server/vulnerabilities/nvd/sync/cve_syncer.go

```diff
@@ -738,6 +1027,284 @@ func convertAPI20CVEToLegacy(cve nvdapi.CVEItem, logger log.Logger) *schema.NVDC
 	}
 }
 
+func convertVulnCheckCVEToLegacy(cve VulnCheckCVE, logger log.Logger) *schema.NVDCVEFeedJSON10DefCVEItem {
```

Comment:
This is mostly the same as `convertAPI20CVEToLegacy`, right? Any way that code can be shared?

Reply:
I did some refactoring to reuse `convertAPI20CVEToLegacy` and pulled out the Configuration Node replacement logic specific to VulnCheck.